src/index.html

```diff
@@ -22,6 +22,399 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        class="header__logo"
+        href="index.html"
+      >
+        <img
+          src="./images/logo.png"
+          alt="logo"
+        />
+      </a>
+
+      <nav class="header__nav">
+        <ul class="header__nav-list">
+          <li class="header__nav-item">
+            <a
+              class="header__nav-link is-active"
+              href="index.html"
+            >
+              Apple
```

Comment:
It is a good practice to use `#` for placeholder links or, if these links are meant to be functional, provide the actual paths. Make sure to update the `href` attribute with the correct path or use `#` if the links are not functional yet. [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules)